layouts/partials/header-logo.html

```diff
@@ -1,19 +1,3 @@
-    </g>
-    <path style="fill: var(--color-body, #3C414B);" d="M346,18 C349.313708,18 352,20.6862915 352,24 L352,52.0843558 C335.814472,61.9012857 325,79.6877153 325,100 C325,120.312285 335.814472,138.098714 352,147.915644 L352,176 C352,179.313708 349.313708,182 346,182 L24,182 C20.6862915,182 18,179.313708 18,176 L18,148.507613 C34.7388911,138.824806 46,120.727489 46,100 C46,79.2725107 34.7388911,61.1751942 18,51.4923868 L18,24 C18,20.6862915 20.6862915,18 24,18 L346,18 Z M336,34 L34,34 L34,43.002 L34.1980271,43.1548511 C51.2773814,56.4369747 61.7571762,76.9083185 61.9958345,99.2209152 L62,100 C62,122.614632 51.4759786,143.408582 34.1980271,156.845149 L34,156.997 L34,166 L336,166 L336,156.209 L335.789156,156.041882 C319.288398,142.728797 309.232976,122.620173 309.003996,100.763155 L309,100 C309,77.8483886 319.096529,57.4260064 335.789156,43.9581181 L336,43.79 L336,34 Z" id="Combined-Shape" fill="var(--color-body)" fill-rule="nonzero"></path>
-    <path style="fill: var(--color-body, #3C414B);" d="M94,145 C98.418278,145 102,148.581722 102,153 L102,170 C102,174.418278 98.418278,178 94,178 C89.581722,178 86,174.418278 86,170 L86,153 C86,148.581722 89.581722,145 94,145 Z M94,103 C98.418278,103 102,106.581722 102,111 L102,132 C102,136.418278 98.418278,140 94,140 C89.581722,140 86,136.418278 86,132 L86,111 C86,106.581722 89.581722,103 94,103 Z M94,61 C98.418278,61 102,64.581722 102,69 L102,90 C102,94.418278 98.418278,98 94,98 C89.581722,98 86,94.418278 86,90 L86,69 C86,64.581722 89.581722,61 94,61 Z M94,19 C98.418278,19 102,22.581722 102,27 L102,48 C102,52.418278 98.418278,56 94,56 C89.581722,56 86,52.418278 86,48 L86,27 C86,22.581722 89.581722,19 94,19 Z" id="Line" fill="var(--color-body)" fill-rule="nonzero"></path>
-    <g style="fill: var(--link-default, #D54716);" id="FIP" transform="translate(127.36, 57.04)" fill="var(--link-default)" fill-rule="nonzero">
-      <polygon id="Path" points="0 0 57 0 57 17.28 18.72 17.28 18.72 34.56 54.12 34.56 54.12 51.84 18.72 51.84 18.72 84.96 0 84.96"></polygon>
-      <polygon id="Path" points="70.2 0 88.92 0 88.92 84.96 70.2 84.96"></polygon>
-      <path d="M105.96,0 L137.52,0 C141.92,0 146.08,0.4 150,1.2 C153.92,2 157.34,3.38 160.26,5.34 C163.18,7.3 165.5,9.94 167.22,13.26 C168.94,16.58 169.8,20.76 169.8,25.8 C169.8,30.76 169,34.92 167.4,38.28 C165.8,41.64 163.6,44.32 160.8,46.32 C158,48.32 154.68,49.74 150.84,50.58 C147,51.42 142.84,51.84 138.36,51.84 L124.68,51.84 L124.68,84.96 L105.96,84.96 L105.96,0 Z M124.68,36 L137.16,36 C138.84,36 140.46,35.84 142.02,35.52 C143.58,35.2 144.98,34.66 146.22,33.9 C147.46,33.14 148.46,32.1 149.22,30.78 C149.98,29.46 150.36,27.8 150.36,25.8 C150.36,23.64 149.86,21.9 148.86,20.58 C147.86,19.26 146.58,18.24 145.02,17.52 C143.46,16.8 141.72,16.34 139.8,16.14 C137.88,15.94 136.04,15.84 134.28,15.84 L124.68,15.84 L124.68,36 Z" id="Shape"></path>
-    </g>
-  </g>
+<svg width="145" height="36">
```

Comment:
```suggestion
<svg>
```

Do we need the width and height on the element? It's already set in the SCSS file :)